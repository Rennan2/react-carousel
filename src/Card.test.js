import { render } from "@testing-library/react";
import Card from "./Card.js";

// smoke test for Card
it("renders without crashing", function () {
    render(<Card
        caption="caption"
        src="imagesource.com"
        currNum="1"
        totalNum="1"
    />);
});

// snapshot test for Card
it("matches snapshot", function () {
    const { asFragment } = render(<Card
        caption="caption"
        src="imagesource.com"
        currNum="1"
        totalNum="1"
    />);
    expect(asFragment()).toMatchSnapshot();
});