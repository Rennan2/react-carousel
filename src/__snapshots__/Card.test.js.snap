// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      caption
    </h4>
    <img
      alt="caption"
      class="Card-image"
      src="imagesource.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 1.
    </small>
  </div>
</DocumentFragment>
`;